_articles/github-for-collaboration/collaborative-workflow-for-contributions.md

```diff
@@ -60,6 +60,10 @@ When you’re writing commit messages, you’ll want to make them short, clear,
     4.  Write a great commit message
     5.  Make a pull request!
 
+![New Pull Request]({{ site.baseurl }}/img/create-pull-req-1.png)
```

Comment:
Maybe the solution is to split step 5 into several steps? Basically adding a few words in between the images that explain what you can see in them.